src/Bundle/BootEnum.php

```diff
@@ -0,0 +1,10 @@
+<?php
+namespace Greg0ire\EnumBundle\Bundle;
+
+class BootEnum extends \Greg0ire\EnumBundle\BaseEnum
+{
+    public static function inject($parameters)
+    {
+        parent::inject($parameters);
+    }
+}
```

Comment:
Good catch @roukmoute ! @flug, please configure your editor to add a "newline" character add the end of each file. vim does that by default, phpstorm can be configured to do so.